packages/client/src/classes/client.js

```diff
@@ -43,51 +31,45 @@ class Client {
     };
   }
 
-  /**
-   * Set API key
-   */
   setApiKey(apiKey) {
     this.apiKey = apiKey;
+
+    if (!this.isValidApiKey(apiKey)) {
+      console.warn(`API key does not start with ${API_KEY_PREFIX}`);
```

Comment:
Here is a spot to put the message: https://github.com/sendgrid/sendgrid-nodejs/blob/master/packages/helpers/constants/index.js -- nit: period at the end.

Reply:
That's in the helpers package. I don't like defining the message in 1 package and using it in another.